.github/workflows/publish.yml

```diff
@@ -43,7 +43,8 @@ jobs:
       - name: Set target repo checkout branch
         # Note: Branches registered here MUST BE protected in the target repo!
         if: |
-          fromJSON(steps.inputs.outputs.result).repo == 'sentry-migr8' && fromJSON(steps.inputs.outputs.result).merge_target == 'tmp-merge-target'
+          fromJSON(steps.inputs.outputs.result).repo == 'sentry-migr8' && fromJSON(steps.inputs.outputs.result).merge_target == 'tmp-merge-target' ||
+          fromJSON(steps.inputs.outputs.result).repo == 'sentry-javascript' && fromJSON(steps.inputs.outputs.result).merge_target == 'v7'
```

Comment:
let's add an `||(newline)false` on the end so the blame stays consistent when adding / removing entries